core-bundle/contao/classes/PurgeData.php

```diff
@@ -102,8 +102,12 @@ public function run()
 			// Get the current folder size
 			foreach ($config['affected'] as $folder)
 			{
+				if (str_contains($folder, '%'))
+				{
+					$folder = StringUtil::stripRootDir($parameterBag->resolveValue($folder));
+				}
+
 				$total = 0;
-				$folder = sprintf($folder, $strCachePath);
```

Comment:
Isn’t this a BC break if someone used this previously like 
`$GLOBALS['TL_PURGE']['folders']['foo']['affected'] = array('%s/foo/bar')`?
Or was this never officially supported?

Reply:
That was never officially supported.